gravitee-apim-portal-webui-next/src/app/catalog/catalog.component.ts

```diff
@@ -13,20 +13,29 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+import { CommonModule } from '@angular/common';
 import { Component } from '@angular/core';
 import { MatCard, MatCardContent } from '@angular/material/card';
 
 import { BannerComponent } from '../../components/banner/banner.component';
+import { CardComponent } from '../../components/card/card.component';
 
 @Component({
   selector: 'app-catalog',
   standalone: true,
-  imports: [BannerComponent, MatCard, MatCardContent],
+  imports: [BannerComponent, MatCard, MatCardContent, CardComponent, CommonModule],
   templateUrl: './catalog.component.html',
   styleUrl: './catalog.component.scss',
 })
 export class CatalogComponent {
-  apis: string[] = [];
+  apis = [
+    {
+      title: 'Test tile',
```

Comment:
Maybe create a class here and add `id` to the ApiVM object?

Reply:
Done